<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Position Based Fluid</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="load_xml_button">
        <property name="text">
         <string>Load XML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xml_path"/>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="import_particles_button">
        <property name="text">
         <string>ImportObj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="obj_path"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QPushButton" name="start_sim_button">
        <property name="text">
         <string>Start/Pause Simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_sim_button">
        <property name="text">
         <string>Export Simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="8">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>900</width>
        <height>720</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
